tool/npm_template/node_wrapper.dart

```diff
@@ -68,29 +63,32 @@ abstract class JSValidationOptions {
 external Exports get exports;
 
 @JS("Object.keys")
-external List<String> getKeys(Object value);
+external List<String> _getKeys(Object value);
 
 void main() {
-  exports.validateBytes = allowInterop((String filename, Uint8List data,
-          GetResourceCallback getResource, JSValidationOptions options) =>
+  exports.validateBytes = allowInterop((Uint8List data, Object options) =>
       new Promise<Object>(allowInterop((resolve, reject) {
-        validateBytes(filename, data, getResource, options).then((result) {
+        validateBytes(data, options).then((result) {
           resolve(jsify(result));
-        }, onError: reject);
+        }, onError: (Object e) => reject(e.toString()));
```

Comment:
Does the client lose access to anything important as a result of hard-coding the `toString` here?

Reply:
For example maybe a stack trace from the non-minified version of this?

Reply:
All properties of that object will be minified, so they won't be really accessible. I'll look into how to provide a stack trace.